src/main/java/com/phonebookservice/util/Util.java

```diff
@@ -0,0 +1,30 @@
+package main.java.com.phonebookservice.util;
+
+import java.util.Collection;
+
+public final class Util {
```

Comment:
I suggest have two utility 
one for StringUtility and other for CollectionUtility